modules/msgpack/src/msgpack_module.cpp

```diff
@@ -0,0 +1,113 @@
+#include <msgpack_module.hpp>
+
+namespace ArkMsgpack
+{
+    CObjekt get_cobjekt(const Value &ark_objekt, ValueType type)
+    {
+        CObjekt objekt;
+
+        if(type == ValueType::NFT)
+        {
+            if(ark_objekt == Ark::True)
+                objekt = true;
+            else if(ark_objekt == Ark::False)
+                objekt = false;
+        }
+        else if(type == ValueType::Number)
+            objekt = static_cast<Value>(ark_objekt).number();
+        else if(type == ValueType::String)
+            objekt = static_cast<Value>(ark_objekt).string_ref();
+        else
+            objekt = static_cast<Value>(ark_objekt).list();
+        return objekt;
+    }
+    Value list_packing(std::vector<Value> &src_list)
```

Comment:
I'm a bit of a maniac, could you add blank lines between a closing brace `}` and a new line?

Reply:
No problem

Reply:
And I also changed **objekt** to **object** for more readability for others developpers